app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package ar.com.wolox.android.example.ui.home.news
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.New
+import javax.inject.Inject
+
+class NewsAdapter @Inject constructor() : RecyclerView.Adapter<NewsHolder>() {
+
+    private var mNewsList: MutableList<New> = ArrayList()
+
+    override fun onCreateViewHolder(viewGroup: ViewGroup, viewType: Int): NewsHolder {
+        val layoutInflater = LayoutInflater.from(viewGroup.context)
+        val itemView = layoutInflater.inflate(R.layout.item_new, viewGroup, false)
+        return NewsHolder(itemView)
+    }
+
+    override fun getItemCount(): Int = mNewsList.size
+
+    override fun onBindViewHolder(holder: NewsHolder, index: Int) {
+        val item = mNewsList[index]
```

Comment:
You can omit this variable. This will also allow you to make an inline function like `override fun onBindViewHolder(holder: NewsHolder, index: Int) = holder.bindHolder(mNewsList[index])`, which I think is more expressive.

Reply:
@leonplondon and this one?